bundle/edu.gemini.lchquery.servlet/src/main/scala/edu/gemini/lchquery/servlet/LchQueryParam.scala

```diff
@@ -94,7 +94,14 @@ object LchQueryParam {
         Option(inv).map(i => s"${i.getFirst} ${i.getLast}").filterNot(_.trim.isEmpty)
 
       val spProg = toSPProg
-      (investigatorName(spProg.getGsaPhase1Data.getPi) :: spProg.getGsaPhase1Data.getCois.asScala.map(investigatorName).toList).
+      val piName = for {
+        f <- Option(spProg.getPIFirstName)
+        l <- Option(spProg.getPILastName)
+        n = s"$f $l".trim
+        if !n.isEmpty
+      } yield n
```

Comment:
Perhaps the test case works most of the time but not for all randomly generated input.
 
````
[error] (bundle_edu_gemini_lchquery_servlet/test:test) sbt.TestsFailedException: Tests unsuccessful
````